statistics/handle_test.go

```diff
@@ -437,3 +437,52 @@ func newStoreWithBootstrap(statsLease time.Duration) (kv.Storage, *domain.Domain
 	do.SetStatsUpdating(true)
 	return store, do, errors.Trace(err)
 }
+
+func (s *testStatsSuite) TestCorrelation(c *C) {
+	defer cleanEnv(c, s.store, s.do)
+	testKit := testkit.NewTestKit(c, s.store)
+	testKit.MustExec("use test")
+	testKit.MustExec("create table t(c1 int primary key, c2 int)")
```

Comment:
When there is a column that is the only part of one index, like `create table t(a int, b int, index idx(a))`, we will not create statistics for the column(Refer https://github.com/pingcap/tidb/blob/master/planner/core/planbuilder.go#L675). Maybe we need to remove that logic.